kite-data/kite-data-core/src/main/java/org/kitesdk/data/spi/DefaultConfiguration.java

```diff
@@ -33,7 +34,19 @@
 public class DefaultConfiguration {
 
   // initialize the default configuration from the environment
-  private static Configuration conf = new Configuration();
+  private static Configuration conf;
+
+  static{
+    //read system property for Oozie Action Configuration XML
+    String oozieActionXml = System.getProperty("oozie.action.conf.xml");
+    if(oozieActionXml != null && oozieActionXml.length() > 0) {
+      Configuration.addDefaultResource(new Path("file:///", oozieActionXml).toString());
```

Comment:
I think that default resources are file paths that are loaded from the classpath, i.e. they shouldn't have a file:// schema.